File: BikeShop.Backend/BikeShop.Payments/BikeShop.Payments.Domain/DTO/Requests/NewBillDTO.cs
﻿using BikeShop.Payments.Domain.DTO.Requests.Payment;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BikeShop.Payments.Domain.DTO.Requests
{
    public class NewBillDTO
    {
        public int ShopId { get; set; }
        public string Description { get; set; }
        public Guid UserId { get; set; }
        public Guid? ClientId { get; set; }

        public List<BillProductDTO> Products { get; set; }
        public AddPaymentDTO Payment { get; set; }
    }
}

File: BikeShop.Frontend/webclient/src/pages/workspace/Cashbox/models/NewBillDTO.ts
import {BillProductDTO} from "./BillProductDTO"

export interface NewBillDTO {
    cash: number
    bankCount: number
    card: number
    personalBalance: number
    userId: string
    clientId: string
    shopId: string
    currencyId: number
    description: string
    products: BillProductDTO[]
}